spotinst_sdk2/clients/ocean_cd/__init__.py

```diff
@@ -813,7 +813,7 @@ def ocean_cd_describe_workloads_active_operations(self, workload_id: str):
 
         return formatted_response["response"]["items"]
 
-    def ocean_cd_describe_workloads_revision(self, workload_id: str):
+    def ocean_cd_describe_workloads_revision(self, workload_id: str, namespace: str, cluster_id: str, kind: str):
```

Comment:
new arguments are not update in the argument list in the docstring

Reply:
updated arguments in the docstring